### ISSUE TITLE
Add pydoctstyle?

### ISSUE THREAD

**Issue by goanpeca** [pybee/beefore#15]

**Comment by freakboy3742** [comment]
Absolutely. This is somewhat related to #6, as well as tasks building Sphinx documentation and making sure there are no errors.

**Comment by goanpeca** [comment]
Cool :-), I can work on this, should this be part of https://github.com/pybee/beefore/blob/master/beefore/checks/pycodestyle.py or a separate linter

https://github.com/pybee/beefore/blob/master/beefore/checks/pydocestyle.py ?

**Comment by goanpeca** [comment]
@freakboy3742 should I add this extension https://pypi.python.org/pypi/flake8-docstrings ? or use pydocstyle directly as a separate linter/check? or ?

**Comment by freakboy3742** [comment]
This was discussed on Gitter; the end point of the discussion was to wrap it into PyCodeStyle (since the easiest implementation is as a flake8 check anyway).
